includes/class-sensei-guest-user.php

```diff
@@ -164,6 +163,51 @@ public function sensei_set_current_user_to_none_if_not_open_course_related_actio
 		}
 	}
 
+	/**
+	 * Initializes the backend and admin actions for Guest users.
+	 *
+	 * @since $$next-version$$
+	 */
+	public static function init_guest_user_admin() {
+		add_filter( 'editable_roles', [ static::class, 'remove_guest_student_role' ], 11 );
+
+		add_action( 'pre_user_query', [ static::class, 'remove_guest_users' ], 11 );
```

Comment:
Naming-wise, `remove_` is a bit misleading here, `hide_` or `filter_out_` could be clearer.

Reply:
Right, changed the names [here](https://github.com/Automattic/sensei/pull/6318/commits/b16d02f63f6b88d3124876c4432ad0775fd657d9) and [here](https://github.com/Automattic/sensei/pull/6318/commits/3a423b710225004e5b6fb1e27c3589ae4d2c96f8)